src/phoenix/core/umap_parameters.py

```diff
@@ -0,0 +1,7 @@
+from dataclasses import dataclass
+from typing import Union, Dict
+
+
+@dataclass
+class UMAPParameters:
+    default_umap_params: Dict[str, Union[int, float]]

```

Comment:
Might add validation here to ensure the values are correct.

Also since this is UMAP specific, it might make more sense to co-locate this with `pointcloud`